src/importexport/audioexport/iaudioexportconfiguration.h

```diff
@@ -35,6 +35,7 @@ class IAudioExportConfiguration : MODULE_EXPORT_INTERFACE
     virtual ~IAudioExportConfiguration() = default;
 
     virtual int exportMp3Bitrate() const = 0;
+    virtual void setExportMp3Bitrate(int bitrate) = 0;
     virtual void setExportMp3Bitrate(std::optional<int> bitrate) = 0;
```

Comment:
Now I think it would be nice to rename this method, to make the difference clear. For example, something like `setExportMp3BitrateOverride` will do.

(And then I'd rename `m_exportMp3Bitrate` too, to `m_exportMp3BitrateOverride`.)

(And the same for the png resolution.)